questions/lookat.md

```diff
@@ -7,7 +7,7 @@ _This method describes how to check whether the player / an entity is **looking
 in Java and later Bedrock versions this can actually be achieved in a single command, thanks to the versatility of the `execute` command. We'll use the following subcommands to achieve our goal:
 
 - `as @a at @s` to modify the execution entity and position.  
-- `anchored eyes` to move the execution position up to the players eyes.  
+- `anchored eyes` to move the execution position up to the players eyes. See this related bug [MC-169665](https://bugs.mojang.com/browse/MC-169665)
```

Comment:
That's a bug for 1.15 and it's marked as invalid? why did you add this here?

If it's for an explanation, it needs more info rather than a link to a bug that implies that something about this isn't working.

Reply:
The command

    /execute anchored eyes run summon cow

Will not work but the command

    /execute anchored eyes positioned ^ ^ ^ run summon cow

will work

Reply:
Yeah, I figured you're referring to that. Mkay how can we change this so it's clearer for the reader? Maybe something like this:

> ... up to the players eyes. For it to take effect, you'll need a `positioned ^ ^ ^` subcommand at some point after this subcommand (which we have 2 subcommands later). See this related bug [MC-169665](https://bugs.mojang.com/browse/MC-169665)

it's a bit verbose, but I think that's better than confusing the user.